views/beers.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <h1>Beers</h1>
+  <div class="row">
+        {{#each beersFromApi}} 
+        <div class="col-md-4">
```

Comment:
Try to comment this code which you already have in the partial and use this:

{{> beersArray this}}